legate/driver/command.py

```diff
@@ -30,18 +30,11 @@
 def cmd_bind(
     config: ConfigProtocol, system: System, launcher: Launcher
 ) -> CommandPart:
-    cpu_bind = config.binding.cpu_bind
-    mem_bind = config.binding.mem_bind
-    gpu_bind = config.binding.gpu_bind
-    nic_bind = config.binding.nic_bind
-
-    if all(x is None for x in (cpu_bind, mem_bind, gpu_bind, nic_bind)):
-        return ()
-
     ranks = config.multi_node.ranks
 
     opts: CommandPart = (
         str(system.legate_paths.bind_sh_path),
+        "--launcher",
         "local"
         if launcher.kind == "none" and ranks == 1
         else str(launcher.kind),
```

Comment:
Note that this can pass "--launcher none", if we're externally launched on multiple ranks/nodes. It doesn't look like this is currently handled in `bind.sh` (it should do what is currently being done in the case of "auto").

Reply:
https://github.com/nv-legate/legate.core/pull/461/commits/6eef4c185b5038fad992352867009e85c7b34c56 if I understand correctly, when the the driver launcher is "none" (i.e. usually unspecified, since that is the default) and there at multi-ranks, we want the "auto" behavior.